gui/src/2D/controls/container.ts

```diff
@@ -98,9 +105,11 @@ export class Container extends Control {
     /**
      * Creates a new Container
      * @param name defines the name of the container
+     * @param renderToIntermediateTexture renders children to an intermediate texture rather than directly to host, useful for alpha blending
      */
-    constructor(public name?: string) {
+    constructor(public name?: string, renderToIntermediateTexture?: boolean) {
         super(name);
+        this._renderToIntermediateTexture = renderToIntermediateTexture ? true : false;

```

Comment:
```suggestion
        this._renderToIntermediateTexture = !!renderToIntermediateTexture;

```

Reply:
I removed this from the constructor, and made it a settable property after construction. Is this perhaps a better approach?